graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js

```diff
@@ -3,6 +3,7 @@ import Reflux from 'reflux';
 import URLUtils from 'util/URLUtils';
 import ApiRoutes from 'routing/ApiRoutes';
 import fetch from 'logic/rest/FetchProvider';
+import fetchPeriodically from 'logic/rest/FetchProvider';
```

Comment:
This is not doing what it should. It uses the default export from `FetchProvider`, which is the `fetch` function, so there is no difference with the original code, except that the function is locally using another name.
In order to fix the issue we should use a named import here `import { fetchPeriodically }...`  to use the right function.